lib/plugins/deploy.js

```diff
@@ -56,7 +56,8 @@ class Deploy {
         },
         commands: {
           function: {
-            usage: 'Deploy a single function from the service',
+            usage:
+              'Deploy a single function from the service. It has limitations - read more here: https://github.com/serverless/serverless/blob/master/docs/providers/aws/cli-reference/deploy-function.md',
```

Comment:
After second though, I would remove that note. I've looked around and we do not practice linking to documentation in `usage` for specific commands + it's a general `function` command that is not only related to AWS provider, which will be confusing to people that use different providers.